lib/SYCL/inSPIRation.cpp

```diff
@@ -235,11 +262,43 @@ struct inSPIRation : public ModulePass {
 
   /// Visit all the functions of the module
   bool runOnModule(Module &M) override {
+    // funcCount is for naming new name for each function called in kernel
+    int funcCount = 0;
```

Comment:
I guess it would not work on some extern functions... We will fix that when we have the problem... :-)